core/src/main/java/com/google/errorprone/bugpatterns/UnusedMethod.java

```diff
@@ -214,7 +215,7 @@ private boolean isMethodSymbolEligibleForChecking(MethodTree tree) {
           }
         }
 
-        return tree.getModifiers().getFlags().contains(Modifier.PRIVATE);
+        return !!ASTHelpers.canBeRemoved(methodSymbol, state);
```

Comment:
```suggestion
        return ASTHelpers.canBeRemoved(methodSymbol, state);
```
(Javascript vibes :stuck_out_tongue:.)